patroni/postgresql.py

```diff
@@ -292,6 +293,11 @@ def reload_config(self, config):
     def pending_restart(self):
         return self._pending_restart
 
+    @staticmethod
+    def configuration_allows_rewind(data):
+        return data.get('Current wal_log_hints setting', 'off') == 'on' \
+            or data.get('Data page checksum version', '0') != '0'
```

Comment:
Yes, my first comment is exactly that we didn't check full_page_writes, but we should.
The second comment dealt with doing various configuration checks in different functions, the PostgreSQL's in configuration_allows_rewind and Patroni's in can_rewind. Wouldn't it make sense to put them all together?